### PR TITLE
Make metrics headings consistently left justified

### LINKED ISSUE #323: Metrics table headings are right-aligned in Firefox
Noticed this while working on #301.  In most browsers the headings are left-aligned but in Firefox they're right-aligned.

### PR DESCRIPTION
**Pull request by RByers**
Use a :not selector to avoid conflicting CSS rules that depend on
variabile specificity rules.  Fixes #323

### CODE REVIEW

File: static/sass/elements/chromedash-metrics.scss [hunk 1]
```diff
@@ -18,6 +18,21 @@
       display: none;
     }
 
+    label {
+      font-weight: 600;
+      // word-wrap: break-word;
```
Comment (ebidel) [inline static/sass/elements/chromedash-metrics.scss]: remove

**Comment by RByers** [comment]
Here's how this looks with this PR applied (and a fake table entry inserted) in Chrome:
image

And Firefox:
image

File: static/sass/elements/chromedash-metrics.scss [hunk 2]
```diff
@@ -18,6 +18,21 @@
...
+    }
+
+    > :first-child {
+      @include flex(1);
+      margin-right: 10px;
+      // max-width: 175px;
```
Comment (ebidel) [inline static/sass/elements/chromedash-metrics.scss]: remove

**Comment by RByers** [comment]
@mdittmer PTAL

File: static/sass/elements/chromedash-metrics.scss (hunk 1 above)
Comment (RByers) [inline static/sass/elements/chromedash-metrics.scss]: Will do (I was just moving the existing rules around - thought the commented out stuff might still carry some significance).

**Comment by ebidel** [comment]
LGTM. Just the nits.

**Comment by RByers** [comment]
Done, also removed the other existing commented-out rule.

**Pull request merged**